### ISSUE TITLE
[1.21] Small QoL and improvements 

### ISSUE THREAD

**Issue by ffuentesm** [klikli-dev/occultism#1155]
Mostly a few things that I noticed when playing the mod that make it a bit frustrating due to that. 

1. <s>Otherwood lacks the `logs_that_burn` tag, which allows it to be used on the charcoal crafting recipe. </s>
2. <s>OtherStone lacks the `c:stones` tag</s>
3. <s>Rename `Otherworld Wood` to `Otherworld Log`</s>
4. ~~Allow for Shift-clicking on the Crusher menu to transfer items to and from the inventory.
(I have a habit of shift-clicking to pull and add items to inventories, like the furnace, so when I try to shift-click the ores to the crusher, is frustrating that it fails)~~
5. ~~Change the Crusher inventory access method.
(The "Shift-Click with empty hand" method is kind of frustrating, Villagers let you access their UI with a simple click, and some interactions that require to Shift click to access the UI let you do it so with items in hand. The empty hand is the main problem since I keep my hot bar full, so needing to open my menu to free a slot adds up with time. Also, this interferes with mods that use the same interaction mechanic, like Carry On.)~~
6. <s>Make the Binding book crafting consistent with the item texture</s>
7. Add a "Wondering Demonic Trader"
(This is more something that came to mind that could expand the use of the 3rd Eye effect. Whenever a wondering trader spawns, there is a small chance it's actually a Demonic trader. Without the goggles or the 3rd eye effect, it acts like a normal wandering trader, but when you have that effect on you, you can see the texture chance, like otherworld blocks, and the trades he gives are occultism related.)
8. ~~Stack Cooking on the demonic partner.
(Allow Demonic Partner to cock the entire stack of food you are holding if you shift-click on them with a stack on hand instead of one item at a time.)~~
9. <s>Allow the Divination Rod to search ancient debris and other ores. </s>
10. <s>Texture change for Purified Ink.</s>
![purified_ink](https://github.com/user-attachments/assets/b56b213f-77a9-41d2-be7a-eb57622159aa)
11. <s>Implement the Item tooltip on the Crusher UI.</s>
![Image](https://github.com/user-attachments/assets/b81ad64c-2324-4e92-a147-e25b652ff2dc)

**Comment by Eqis-Edu** [comment]
Item 3 done in #1201 and #1204

**Comment by pixelated-pup** [comment]
I was looking into item 4, and from what I can tell this does work, at least on current version. Could you double check if it also works for you?

**Comment by ffuentesm** [comment]
@ThePixelatedCat yes and not. 
The shift-Rclicking does not work if the item is in my hot bar. 
![Image](https://github.com/user-attachments/assets/aba0d2e5-d580-4d44-b299-6b534f611b9f)

When I do it, it forces the item to go to the first slot on my inventory 
if there is an item there already, nothing happens. 
![Image](https://github.com/user-attachments/assets/c0935ecd-2508-40b0-a8b9-ba90503d245b)

The item will only be transfer to the Crusher slot, if I shift-Rclick on the item, when the item is on the first inventory slot. 

![Image](https://github.com/user-attachments/assets/8a151e0a-0cfd-4959-8c45-1259bfb22e1e)

Video of problem: https://github.com/user-attachments/assets/1dea5205-5e64-4a14-81c0-a06e1ca01390

**Comment by pixelated-pup** [comment]
Yep, I noticed this when I was testing. Didn't think about the fact that nothing happens if there's an item already there, certainly makes it more problematic. I'm looking into it currently and I believe I've figured out the issue

**Comment by klikli-dev** [comment]
Item 4 and 5 done in https://github.com/klikli-dev/occultism/issues/1155

**Comment by pixelated-pup** [comment]
Item 8 done in PR #1302

**Comment by pixelated-pup** [comment]
Should the wandering demonic trader be moved to the feature tracker, given it's a relatively major addition?

**Comment by ffuentesm** [comment]
I think so. Then you can mark this as completed

**Comment by Eqis-Edu** [comment]
11 implemented in #1341 

![Image](https://github.com/user-attachments/assets/ad1c0a66-b61c-4ba5-9cef-355bc801e51c)

**Comment by teabaginc** [comment]
The processing demons should have a clear way to distinguish them from crusher/smelter/crystallizer. 
They currently all look alike, and it's annoying to have to check on them.

**Comment by ffuentesm** [comment]
> The processing demons should have a clear way to distinguish them from crusher/smelter/crystallizer. They currently all look alike, and it's annoying to have to check on them.

My suggestion is to Either:
- Use the same model to represent the demon tier, and have different colors to represent each type. 
- Have different models to differentiate the type, and keep them all the same color to represent their tier. 
- Have them all be the same, just give each type a different prop item to differentiate the type.

**Comment by Eqis-Edu** [comment]
Finishing all items with 7 in #1465

**Comment by klikli-dev** [comment]
Closing this as complete
